add blog post and minor changes to whatsnew (#106407)
Azure/azure-content: articles/cognitive-services/Computer-vision/whats-new.md

## March 2023

> ### Microsoft Improves Computer Vision Services with groundbreaking foundation model

Reviewer: ```suggestion
### Computer Vision Image Analysis 4.0 SDK public preview
```

## March 2023

### Computer Vision Image Analysis 4.0 SDK public preview

> The groundbreaking [Florence foundation model](https://www.microsoft.com/en-us/research/project/projectflorence/) has now been integrated into Azure Computer Vision. The improved Vision Services enable developers to create cutting-edge, market-ready, responsible computer vision applications across various industries. Customers can now seamlessly digitize, analyze, and connect their data to natural language interactions, unlocking powerful insights from their image and video content to support accessibility, drive acquisition through SEO, protect users from harmful content, enhance security, and improve incident response times. Learn more [here](https://aka.ms/florencemodel).

Reviewer 2: ```suggestion

The [Florence foundation model](https://www.microsoft.com/en-us/research/project/projectflorence/) is now integrated into Azure Computer Vision. The improved Vision Services enable developers to create market-ready, responsible computer vision applications across various industries. Customers can now seamlessly digitize, analyze, and connect their data to natural language interactions, unlocking powerful insights from their image and video content to support accessibility, drive acquisition through SEO, protect users from harmful content, enhance security, and improve incident response times. For more information, see [Announcing Microsoft's Florence foundation model](https://aka.ms/florencemodel).
```

changes to adhere to MS Style - downplaying marketing language (doesn't belong in a technical article) and fixing link text (for SEO improvements)
